src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java

```diff
@@ -77,58 +83,59 @@ public boolean getHttpsEnabled() {
         return this.httpsEnabled;
```

Comment:
no "this"?

Reply:
I think going forward we should use this, for all member variables to make it easier to read.